configure.ac

```diff
@@ -90,7 +90,7 @@ AC_ARG_ENABLE([force-CXX98-compat],
     [AS_HELP_STRING([--enable-force-CXX98-compat], [force C++98 build [default=disabled]])])
 if test "x$enable_force_CXX98_compat" = "xyes"; then
     AC_LANG_PUSH([C++])
-    AX_CHECK_COMPILE_FLAG([-std=gnu++98], [CXXFLAGS+=" -std=gnu++98"], [])
+    AX_CHECK_COMPILE_FLAG([-std=c++98], [CXXFLAGS+=" -std=c++98"], [])
```

Comment:
I'm not entirely convinced about the c11, but maybe that's close enough to gnu11, and it should be well tested as it's the default. But please do not change gnu++98/gnu99, as that's fallback for very old platforms that are quite fragile. It might be that on modern compilers they are the same, but that's not the case on ancient gcc versions and so on, which are very difficult to test.

Reply:
Should I retain gnu++98 in the CMake build as well?

Reply:
yes, please

Reply:
Done and pushed. I rephrased the commit to better articulate how we're only using standard C11 and C++11 but older compilation modes retain GNU extensions to ensure we don't break with old compilers.